### PR TITLE
feature(ui): NumberField with a separate icons view

### PR DESCRIPTION
**Pull request by allfayn**
New type for a view plus/minus buttons

image
image

### CODE REVIEW

File: packages/ui/__stories__/TextField.stories.module.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+@use '../styles/constants' as c;
+
 .field {
   margin-bottom: 16px;
 }
+
+.shortField {
+  width: c.$grid * 25;
```
Comment (Sodik) [inline packages/ui/__stories__/TextField.stories.module.scss:8]: I think it should be done on parent element in app not in hive. What if I have to use it with bigger numbers (billions e.t.c)?

**Review by Sodik — CHANGES_REQUESTED** [review]

File: packages/ui/__stories__/NumberField.stories.tsx [hunk 2]
```diff
@@ -33,7 +33,9 @@ const Template: Story<NumberFieldProps> = ({ value: initValue, ...args }) => {
   return (
     <>
       <NumberField {...args} value={value} onChange={setValue} />
+      <NumberField {...args} value={value} onChange={setValue} iconPosition="together" className={storyStyles.shortField} />
```
Comment (fantkolja) [inline packages/ui/__stories__/NumberField.stories.tsx:36]: @allfayn @pawelklasa Should we do something with placeholder?
image

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: packages/ui/__stories__/TextField.stories.module.scss (hunk 1 above)
Comment (allfayn) [reply packages/ui/__stories__/TextField.stories.module.scss:8]: It's not in Hive, it's only for presentation (at Story)

**Review by fantkolja — APPROVED** [review]

**Review by Sodik — APPROVED** [review]

File: packages/ui/__stories__/NumberField.stories.tsx (hunk 2 above)
Comment (allfayn) [reply packages/ui/__stories__/NumberField.stories.tsx]: removed placeholders for this kind of fields at story

**Pull request merged**